h/search/core.py

```diff
@@ -51,6 +51,7 @@ def __init__(
             query.DeletedFilter(),
             query.AuthFilter(request),
             query.GroupFilter(request),
+            query.RepliesFilter(),
```

Comment:
This worked before because `KeyValueMatcher` is just a fallback for any key that is not handled by any other "Matcher".

Reply:
That's correct. The `KeyValueMatcher` is the default, but only works on singular key value pairs.